src/workers/cdc/common.ts

```diff
@@ -8,3 +24,86 @@ export const getTableName = <Entity>(
   con: DataSource,
   target: EntityTarget<Entity>,
 ): string => con.getRepository(target).metadata.tableName;
+
+export const notifyPostContentUpdated = async ({
+  con,
+  post,
+}: {
+  con: DataSource;
+  post: ChangeObject<Post>;
+}): Promise<void> => {
+  const [source, keywords, relatedPosts] = await Promise.all([
+    con.getRepository(Source).findOneBy({
+      id: post.sourceId,
+    }),
+    con.getRepository(PostKeyword).find({
+      where: {
+        postId: post.id,
+      },
+      order: {
+        keyword: 'ASC',
+      },
+    }),
+    con.getRepository(PostRelation).find({
+      where: [
+        {
+          postId: post.id,
+        },
+        {
+          relatedPostId: post.id,
+        },
+      ],
+      order: {
+        createdAt: 'ASC',
+      },
+    }),
+  ]);
+  const articlePost = post as ChangeObject<ArticlePost>;
+  const freeformPost = post as ChangeObject<FreeformPost>;
+
+  const contentUpdatedMessage = new ContentUpdatedMessage({
```

Comment:
Don't we also need the `steps` here?
(not sure what we called them)

Reply:
what steps, for scraping?

Reply:
It should be in `content_meta`, @vpol do you know if one of the fields in [schema](https://github.com/dailydotdev/schema/blob/main/daily-api/pubsub/content-updated.proto) represent it?